doc/agama-security.md

```diff
@@ -0,0 +1,34 @@
+Agama Concepts
+==============
+
+Agama's functionality is divided into backend and frontend. Communication between two parts is done through REST api. Most of the api requires an authorization.
+
+Authorization
+-------------
+
+Authorization is done via password. To get authorized user is asked for a password of root on backend's machine. The password is validated through PAM [1]. Once the authorization succeeds, backend generates an authorization token and passes it back to frontend / user. Agama uses JWT as authorization token [2]. All subsequent calls to the API has to be done together with the token.
```

Comment:
What "To get authorized user" means? There is not another user apart from the root.